src/app/public/modules/modal/modal.component.scss

```diff
@@ -148,3 +153,39 @@
   min-height: 460px;
   margin: -$sky-margin-plus-half;
 }
+
+@include sky-theme-modern {
+  .sky-modal-header {
+    border: none;
+    padding: $sky-theme-modern-space-lg $sky-theme-modern-space-xl;
+  }
+
+  .sky-modal-btn-help,
```

Comment:
Why are we hiding the help button?

Reply:
Modern theme modal use cases do not include the help button.

Reply:
🤯